TWTValidation/JSON Validator/AST Nodes/TWTJSONSchemaDependencyASTNode.m

```diff
@@ -0,0 +1,46 @@
+//
+//  TWTJSONSchemaDependencyASTNode.m
+//  TWTValidation
+//
+//  Created by Jill Cohen on 12/16/14.
+//  Copyright (c) 2014 Two Toasters, LLC.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <TWTValidation/TWTJSONSchemaDependencyASTNode.h>
+
+
+@implementation TWTJSONSchemaDependencyASTNode
+
+- (instancetype)initWithKey:(NSString *)key propertySet:(NSArray *)propertySet
```

Comment:
Can you look over this initializer? My intention is that the node should EITHER have a key/valueNode OR a key/propertySet pair.